### PR TITLE
[Stack Monitoring] add kibana_stats version alias to -mb template

### PR DESCRIPTION
**Pull request by klacabane**
### Summary
Rel https://github.com/elastic/kibana/pull/125315

Add a missing alias to the kibana monitoring template

### Testing
- started an ES with `gradlew run -Drun.license_type=trial`
- verified `kibana-mb` template was created
- started metricbeat monitoring and inspected `kibana_stats.kibana.version`

### CODE REVIEW

File: x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java [hunk 1]
```diff
@@ -78,7 +78,7 @@
      * writes monitoring data in ECS format as of 8.0. These templates define the ECS schema as well as alias fields for the old monitoring
      * mappings that point to the corresponding ECS fields.
      */
-    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id;
+    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_2_0.id;
```
Comment (klacabane) [inline x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java:81]: Need to be careful and update the backported PRs to assign the correct version

Comment (jbaiera) [reply x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java:81]: Discussed on slack, but keeping here for visibility:

Hmm, we should be careful here. This is used to determine if we need to upgrade templates that are present. We probably shouldn't have set this to the product version, but that was the historical approach for monitoring indices that was already in place, so we have been somewhat locked in.

Check out the StackTemplateRegistry file in the stack plugin. That has a version of `1` right now. This version number needs to monotonically increase every update to the templates, and the version number should be the same for identical mappings.

```suggestion
    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id + 1;
```

**Review by jbaiera — APPROVED** [review]
LGTM pending version comment

**Comment by jbaiera** [comment]
@elasticmachine run elasticsearch-ci/part-2

**Comment by klacabane** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.1|[](https://github.com/elastic/elasticsearch/pull/84016)|
|✅|8.0|[](https://github.com/elastic/elasticsearch/pull/84017)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**